lib/isolate_worker.dart

```diff
@@ -142,15 +142,15 @@ class IsolateWorkerPool<T, R> {
     return worker.send(task);
   }
 
-  Future<List<R>> sendMany(List<IsolateTask<T, R>> tasks) async {
+  Future<List<R?>> sendMany(List<IsolateTask<T, R>> tasks) async {
     // return Future.wait(tasks.map((t) => send(t)));
-    List<Future<R>> futures = List(tasks.length);
-    int i = 0;
+    List<Future<R?>?> futures = List<Future<R>?>.filled(tasks.length, null);
+    var i = 0;
 
     // int done = 0;
     while (i < tasks.length) {
       var task = tasks[i];
-      int j = 0;
+      var j = 0;
       while (_workers[j].isTooBusy) {
         j++;
         if (j >= count) {
```

Comment:
Gah, I can't suggest or even comment on lines that are "omitted" by github, but the line just below this line has `Future<Null>` that I'd change to `Future<void>` if I could. It's just a nit but if you have the code checked out and in the process of editing, I'd appreciate that.